src/app/routes/onDemandRadio/getInitialData/index.js

```diff
@@ -82,6 +91,11 @@ export default async ({ path: pathname, pageType, service, toggles }) => {
         ),
         episodeAvailability: getEpisodeAvailability(json),
         radioScheduleData: getRadioScheduleData(json),
+        recentEpisodes: processRecentEpisodes(json, {
```

Comment:
what is the reason for 2 arguments?

Reply:
I believe it was to separate the required argument and the optional params